downloads/swan-lake-release-notes/swan-lake-preview6/RELEASE_NOTE.md

```diff
@@ -0,0 +1,354 @@
+
+#### Standard Library
+
+##### Revamp File and Filepath Modules
+
+The `File` and `Filepath` APIs have been revamped in this release. A summarized list of the changes done is as follows.
+
+1. The `File` and `Filepath` modules have been merged together. There will be no `Filepath` module anymore.
+
+2. Some APIs have been removed.
+
+    - From `filepath` - `extension`, `getPathSeparator`, `getPathListSeparator`, `isReserved`, and `matches`
+    - From `file` - `exists`, `tempDir`
+
+3. The implementation of some APIs has been modified.
+
+    - From `Filepath` - `normalize`
+    - From `File` - `createDir`, `remove`, `copy`, `readDir`, `getMetaData` (earlier `getFileInfo`)
+
+4. Some APIs have been introduced newly.
+
+    - `createTemp` - to create a temporary file in either the default `temp` directory of the OS or in a specific directory.
+    - `createTempDir` - to create a temporary directory as above.
+    - `test` - checks if a file/directory exists, readable, writable, etc.
+
+##### Improved IO Module
+
+The new high-level APIs below were introduced to provide I/O operations.
+
+1. Added a new set of APIs to read from files supporting different data types.
+    
+    E.g., `io:fileReadBytes`, `io:fileReadJson`, `io:fileReadCsv`, and `io:fileReadXml`
+
+2. Added a new set of APIs to read files as streams.
+    
+    E.g., `io:fileReadBlocksAsStream`, `io:fileReadLinesAsStream`, and `io:fileReadCsvAsStream`
+
+#### Language Server
+
+1. Introduced a new code-action `Add Type Cast` to add a typecast when variable assignment fails due to incompatible types.
+
+    **Before execution:**
+    ```ballerina
+    int myInt = 1.1;
+    int myInt = getFloat();
+    ```
+
+    **After execution:**
+    ```ballerina
+    int myInt = <int>1.1;
+    int myInt = <int>getFloat();
+    ```
+
+2. The code-action extension APIs have been revamped in this release. A summarized list of changes is as follows.
+
+    - The `MatchedNode`, `matchedSymbol`, and `matchedExprType` for the cursor position are available now through the `PositionDetails` construct in the `CodeActionContext`.
+    - Listing the priorities can be set for the code-actions now (lower the number, greater the priority).
+
+#### Code to Cloud
+
+1. Introduced code to cloud (c2c) to simplify the experience of developing and deploying Ballerina code in the cloud. Code to cloud builds the containers and required artifacts by deriving the required values from the code without using any annotations.
+
+        ```ballerina
+        import ballerina/http;
+        import ballerina/c2c as _;
+
+        service hello on new http:Listener(9090) {
+            resource function sayHello(http:Caller caller, http:Request req) returns error? {
+                check caller->respond("Hello, World!");
+            }
+        }
+
+        ```
```

Comment:
The above code block is not properly displayed.

Reply:
We need 2 tabs to get code snippets displayed properly on the website although that doesn't look proper in this GitHub view.

They will be fine once they are rendered on the website.